### PR TITLE
feat: Adding login button in topbar

### LINKED ISSUE #279: [FEATURE] Add Login/Register button in navbar
### Description

**Issue 1**
I just experienced something, which is adding a Login/Register button in navbar might lead to great user experience, because when visitor comes to the site and open projects tab, then if they want to Submit their project they have go back to homepage, and login from there, its kinda long and boring step, so I think that adding Register button in navbar would be nice.

**Issue 2**
I found a small typo, here
image
It would be better if it gets fixed :)

### PR DESCRIPTION
**Pull request by rupali-codes**
### Fixes Issue
Closes #279 

### Changes proposed
1. created component LoginButton
2. added this component into topbar.tsx
3. added a login button into sidebar

### Note to reviewers
Let me know for any kind of changes if needed 

### Screenshots
**Desktop**
image

**Other than desktop**
image

### CODE REVIEW

**Review by rohitdasu — APPROVED** [review]

File: components/Topbar/Sidebar.tsx [hunk 1]
```diff
@@ -120,6 +127,18 @@ export const Sidebar = () => {
                 Submit project
                 <MdPostAdd className="text-2xl" />
               </button>
+              {session === null && (
+                <Button
+                  onClick={handleModal}
+                  isDisabled={false}
+                  className="text-md w-full py-3 px-6 font-semibold md:w-auto md:text-lg"
+                >
```
Comment (rohitdasu) [inline components/Topbar/Sidebar.tsx:135]: just a change request, could you make this login item in sidebar similar to other sidebar buttons?

Comment (rupali-codes) [reply components/Topbar/Sidebar.tsx:135]: sure

Comment (rupali-codes) [reply components/Topbar/Sidebar.tsx:135]: @rohitdasu are you talking about color or the style format?

Comment (rohitdasu) [reply components/Topbar/Sidebar.tsx:135]: both, it should match with other items.

Comment (rohitdasu) [reply components/Topbar/Sidebar.tsx:135]: @rupali-codes

Comment (rupali-codes) [reply components/Topbar/Sidebar.tsx:135]: alright

Comment (rupali-codes) [reply components/Topbar/Sidebar.tsx]: done @rohitdasu

Comment (rohitdasu) [reply components/Topbar/Sidebar.tsx]: thanks @rupali-codes great job

**Pull request merged**